modules/gdscript/gdscript_parser.cpp

```diff
@@ -3758,6 +3758,33 @@ bool GDScriptParser::export_annotations(const AnnotationNode *p_annotation, Node
 					return false;
 				}
 				break;
+			case GDScriptParser::DataType::CLASS:
+				// can assume type is a global GDScript class.
+				if (!ClassDB::is_parent_class(export_type.native_type, Resource::get_class_static())) {
```

Comment:
```suggestion
				if (!ClassDB::is_parent_class(export_type.native_type, SNAME("Resource"))) {
```
For consistency with the existing code.

Reply:
I can definitely make this change, but it also kinda confuses me why this would be preferable implementation-wise (if addressed, it'd be in a totally different proposal/PR probably).

I get that using the cached StaticCString is better here, but if that's the case, why not arrange things so that `get_class()` and `get_class_static()` return a `StringName` that is likewise constructed from `_scs_create(...)`, that way you can get performance benefits of `SNAME(...)` built into those methods? That would then allow you to clean up all of these stringly-typed class name references throughout the code.